### ISSUE TITLE
Spacing on items

### ISSUE THREAD

**Issue by indianpoptart** [gabrielemariotti/changeloglib#36]
When I make a new activity, after each item there is a large gap between one item and the next

**Comment by gabrielemariotti** [comment]
Duplicate #28

**Comment by indianpoptart** [comment]
thx, it fixed my issue

for anyone looking at this with the same issue, this helped

While I am trying to fix the #31 I've fix the 23.2.0 issue with a snapshot:

```
You can now download it from Sonatype's snapshots repo.

Add the snapshots repo to your build.gradle

repositories {
    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
}
Add build dependency

dependencies {
    compile 'com.github.gabrielemariotti.changeloglib:changelog:2.0.1-SNAPSHOT'
}
```
